Makefile

```diff
@@ -264,46 +263,6 @@ clean: ## Clean artifacts
 		docs/build
 	make -C docs clean
 
-.PHONY: libpodimage
-libpodimage: ## Build the libpod image
-	${CONTAINER_RUNTIME} build -t ${LIBPOD_IMAGE} .
-
-.PHONY: dbuild
-dbuild: libpodimage
-	${CONTAINER_RUNTIME} run --name=${LIBPOD_INSTANCE} --privileged -v ${PWD}:/go/src/${PROJECT} --rm ${LIBPOD_IMAGE} make all
-
-.PHONY: dbuild-podman-remote
-dbuild-podman-remote: libpodimage
-	${CONTAINER_RUNTIME} run --name=${LIBPOD_INSTANCE} --privileged -v ${PWD}:/go/src/${PROJECT} --rm ${LIBPOD_IMAGE} $(GOBUILD) -ldflags '$(LDFLAGS_PODMAN)' -tags "$(REMOTETAGS)" -o bin/podman-remote ./cmd/podman
-
-.PHONY: dbuild-podman-remote-darwin
-dbuild-podman-remote-darwin: libpodimage
-	${CONTAINER_RUNTIME} run --name=${LIBPOD_INSTANCE} --privileged -v ${PWD}:/go/src/${PROJECT} --rm ${LIBPOD_IMAGE} env GOOS=darwin $(GOBUILD) -ldflags '$(LDFLAGS_PODMAN)' -tags "${REMOTETAGS}" -o bin/podman-remote-darwin ./cmd/podman
-
-.PHONY: test
-test: libpodimage ## Run tests on built image
```

Comment:
We can't ditch `make test` - if we want to remove the dockerized version, we should convert this to run the full suite locally.

Reply:
Okay, I can implement this as a PHONY for all the `local*` targets.